### PR TITLE
Add support for several NTS settings.

### PR DESCRIPTION
**Pull request by Heidistein**
Co-authored-by: Tim Meusel <tim@bastelfreak.de>

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -272,6 +290,14 @@
   Boolean $rtconutc                                                = false,
   Variant[Hash,Array[String]] $hwtimestamps                        = [],
   Optional[Stdlib::Unixpath] $dumpdir                              = undef,
+  Optional[Stdlib::Absolutepath] $ntsserverkey                     = undef,
+  Optional[Stdlib::Absolutepath] $ntsservercert                    = undef,
+  Optional[Integer] $ntsport                                       = undef,
```
Comment (root-expert) [inline manifests/init.pp:295]:
```suggestion
  Optional[Stdlib::Port] $ntsport                                  = undef,
```

File: manifests/init.pp [hunk 2]
```diff
@@ -272,6 +290,14 @@
...
+  Optional[Integer] $maxntsconnections                             = undef,
+  Optional[Integer] $ntsprocesses                                  = undef,
```
Comment (root-expert) [inline manifests/init.pp:297]:
```suggestion
  Optional[Integer[0]] $maxntsconnections                          = undef,
  Optional[Integer[0]] $ntsprocesses                               = undef,
```

File: manifests/init.pp [hunk 3]
```diff
@@ -272,6 +290,14 @@
...
+  Optional[Stdlib::Absolutepath]  $ntsdumpdir                      = undef,
+  Optional[String]  $ntsntpserver                                  = undef,
+  Optional[Integer] $ntsrotate                                     = undef,
```
Comment (root-expert) [inline manifests/init.pp:300]:
```suggestion
  Optional[Integer[0]] $ntsrotate                                  = undef,
```

File: templates/chrony.conf.epp [hunk 4]
```diff
@@ -163,6 +163,46 @@ maxupdateskew <%= $chrony::maxupdateskew %>
...
+<% if $chrony::ntsntpserver { -%>
+
+# https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html#ntsntpserver
+ntsntpserver <%= $chrony::ntsntpserver %>
+<% } -%>
+<% if $chrony::ntsrotate { -%>
+
+# https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html#ntsrotate
+ntsrotate <%= $chrony::ntsrotate %>
+<% } -%>
```
Comment (root-expert) [inline templates/chrony.conf.epp:205]: Minor, these settings were added on Chrony 4.0 as far as I can see. Can you update the links to the documentation?

Comment (Heidistein) [reply templates/chrony.conf.epp]: copy paste effect

**Review by bastelfreak — APPROVED** [review]

**Comment by bastelfreak** [comment]
thanks for the work!

**Pull request merged**